crypto/gopenpgp.go

```diff
@@ -1,14 +1,21 @@
 // Package crypto provides a high-level API for common OpenPGP functionality.
 package crypto
 
+import "sync"
+
 // GopenPGP is used as a "namespace" for many of the functions in this package.
 // It is a struct that keeps track of time skew between server and client.
 type GopenPGP struct {
 	latestServerTime int64
 	generationOffset int64
+	lock             *sync.Mutex
 }
 
-var pgp = GopenPGP{}
+var pgp = GopenPGP{
+	latestServerTime: 0,
+	generationOffset: 0,
+	lock: &sync.Mutex{},
```

Comment:
I would rather use a `sync.RWMutex` (https://pkg.go.dev/sync#RWMutex) since we have that the memory model is consistent with multiple parallel reads :)